client/src/app/site/agenda/components/agenda-list/agenda-list.component.ts

```diff
@@ -77,4 +78,37 @@ export class AgendaListComponent extends ListViewBaseComponent<ViewItem> impleme
     public onPlusButton(): void {
         this.router.navigate(['topics/new'], { relativeTo: this.route });
     }
+
+    /**
+     * Handler for deleting multiple entries. Needs items in selectedRows, which
+     * is only filled with any data in multiSelect mode
+     */
+    public deleteSelected(): void {
+        this.selectedRows.forEach(vm => {
+            this.repo.delete(vm);
+        });
+    }
+
+    /**
+     * Sets multiple entries' open/closed state. Needs items in selectedRows, which
+     * is only filled with any data in multiSelect mode
+     * @param closed true if the item is to be considered done
+     */
+    public setClosedSelected(closed: boolean): void {
+        this.selectedRows.forEach(vm => {
+            this.repo.update({closed: true}, vm);
```

Comment:
closed would be "true" all the time. Perhaps it should be:
`{closed: closed}`

Furthermore, this might use either `await` or .then
I would prefer .then and also catch errors using raiseError.
Same to the other functions that use async functions from the repo